mobile/src/main/java/ch/epfl/sdp/mobile/ui/Graphics.kt

```diff
@@ -29,51 +26,6 @@ val PawniesIcons.SectionSocial
 val PawniesIcons.SectionSettings
   get() = Icons.Default.Settings
 
-val PawniesIcons.Add
```

Comment:
Removing these icons breaks the app, doesn't it ?

Reply:
Yes I corrected it